check_emoji_sequences.py

```diff
@@ -51,7 +51,7 @@ def seq_name(seq):
     def strip_vs_map(seq_map):
```

Comment:
update header comment to indicate intent to pass Android linter and reference the public url of the android linter

Reply:
Done, what does this have to do with line 51? 😝

Reply:
Err. I think I clicked the wrong line lol.